core/src/main/java/io/grpc/EquivalentAddressGroup.java

```diff
@@ -60,6 +61,33 @@ public EquivalentAddressGroup(SocketAddress addr) {
   }
 
   /**
+   * Converts list of ResolvedServerInfoGroup objects into one EquivalentAddressGroup object.
+   */
+  public static EquivalentAddressGroup fromResolvedServerInfoGroups(
```

Comment:
Actually, `GrpclbLoadBalancer` uses this too and stores flat structure internally too.